rdtools/test/analysis_chains_test.py

```diff
@@ -196,12 +223,23 @@ def test_filter_components_no_filters(sensor_parameters):
     assert rd_analysis.sensor_filter_components.empty
 
 
+def test_aggregated_filter_components_no_filters(sensor_parameters):
+    rd_analysis = TrendAnalysis(**sensor_parameters, power_dc_rated=1.0)
+    rd_analysis.filter_params = {}  # disable all index-based filters
+    rd_analysis.aggregated_filter_params = {}  # disable all daily filters
```

Comment:
this should be `filter_params_aggregated`

Reply:
Good catch @cdeline, addressed in https://github.com/NREL/rdtools/pull/348/commits/85886d11d5007bd686575f27e14981ea996a1b5f